### PR TITLE
N43 hexx

### PR DESCRIPTION
**Pull request by ManevilleF**
_Part of #1287_

### CODE REVIEW

File: content/news/043/index.md [hunk 1]
```diff
@@ -75,6 +75,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [hexx]
+
+![hexx 2d example](hexx.png)
+_2d Hex grid example_
+
+An hexagonal tools library made by [@ManevilleF](https://linktr.ee/ManevilleF):
```
Comment (AngelOnFira) [inline content/news/043/index.md:83]: If we can move the links to the bottom, that would be better :)

File: content/news/043/index.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- Manipulate hexagonal coordinates, draw rings, lines, weges, etc
+- Generate hexagonal grids, with conversion between your world and the hexagonal
+coordinates system
+- Compute 3d meshes for your hexagons
+
+[hexx] is engine agnostic, but was made with [bevy] integration in mind and
+provides 2d and 3d
+[examples](https://github.com/ManevilleF/hexx/tree/main/examples)
+
+_Discussions: [Twitter](https://twitter.com/ManevilleF/status/1625159292490854400)
```
Comment (AngelOnFira) [inline content/news/043/index.md:94]: Here as well for moving links

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Comment by AngelOnFira** [comment]
Also, if we can reduce image sizes as well, converting to jpg should work alright :)

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

Great lib, btw - I'll probably migrate Zemeroth to it once I return to development :D

**Pull request merged**